src/main/java/bdv/fx/viewer/project/VolatileHierarchyProjectorPreMultiply.java

```diff
@@ -251,54 +251,54 @@ public boolean map(final boolean clearUntouchedTargetPixels)
 					if (interrupted.get())
 						return null;
 
-					final RandomAccess<ARGBType> targetRandomAccess = target.randomAccess(target);
-					final Cursor<ByteType>       maskCursor         = Views.iterable(mask).cursor();
-					final RandomAccess<A>        sourceRandomAccess = sources.get(iFinal).randomAccess(sourceInterval);
-					int                          myNumInvalidPixels = 0;
-
-					final long[] smin = new long[n];
-					System.arraycopy(min, 0, smin, 0, n);
-					smin[1] = myMinY;
-					sourceRandomAccess.setPosition(smin);
-
-					targetRandomAccess.setPosition(min[0], 0);
-					targetRandomAccess.setPosition(myMinY, 1);
+						final RandomAccess<ARGBType> targetRandomAccess = target.randomAccess(target);
```

Comment:
indentation wrong?

Reply:
I can clean this up if you'd like. I actually looked into this, thinking I had maybe formatted this with the new style schema; It looks not though, the commit for change was from Igor's initial WIP Java 13 commit (https://github.com/saalfeldlab/paintera/commit/a2185e0bd54fb8439710317f5dec701133c32811#diff-72981ce0efa82646cc71a50e8d665ea8341f3d5b4b1ff3f51b30a313384a23e6L267). Would you like me to undo the format changes, so we can tackle them all at the same time?